packages/extension-countdown/package.json

```diff
@@ -28,6 +28,7 @@
     "directory": "packages/extension-countdown"
   },
   "author": "Shaobin Jiang",
+  "authorGitHub": "https://github.com/Shaobin-Jiang",
```

Comment:
I think we should make this compliant with how npm expects it:

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#people-fields-author-contributors